src/ui_parts/about_menu.tscn

```diff
@@ -474,4 +475,7 @@ focus_mode = 0
 mouse_default_cursor_shape = 2
 text = "Close"
 
+[node name="Dropdown" parent="." instance=ExtResource("7_xb47m")]
+layout_mode = 2
+
```

Comment:
Seems like a mistake?

Reply:
Yes, I was testing something earlier and forgot to clean it up.